### DISCUSSION TITLE
Hi Dmitry

### DISCUSSION THREAD

**Discussion by holybora** [pvoid/AndroidBlueprintPlugin#19] [Q&A] [answered]
Hey and Hi 
My name is Slava and I am Android dev, nice to meet you!

For now my project is system app for AOSP and your plugin looks great to start working with it but how should I install it in Intelij IDEA. I work in Intelij IDEA Com edition 2021.3. Is it possible? 

Sorry if my question looks stupid, AOSP looks a little deferent then old plain apps dev.

**Comment by pvoid** [comment, marked as answer]
It's in the JetBrain plugins market but in the alpha channel. To install it open Plugins section in Settings and select "Manage plugin repositories" in the dropdown list behind the gear icon.

Add the channel 'https://plugins.jetbrains.com/plugins/alpha/16759' there and press Ok

Search for the plugin by name.

**Reply by holybora** [reply]
Ow, thanks!!!
Should I use IDEA alpha version for it?

**Reply by pvoid** [reply]
No. The release one goes perfect. It's the plugin that is in alpha.
